app/service/rest.py

```diff
@@ -946,6 +948,25 @@ def check_if_reply_to_address_already_in_use(service_id, email_address):
         )
 
 
+@service_blueprint.route('/<uuid:service_id>/returned-letter-statistics', methods=['GET'])
+def returned_letter_statistics(service_id):
+
+    most_recent = fetch_most_recent_returned_letter(service_id)
```

Comment:
This will return dates that are longer than 7 days ago. If it does this then we will still run the `fetch_returned_letter_count` function which will be of no use as there won't be anything in the last 7 days so a wasted DB query

Reply:
I’ve removed the wasted query in 8c47d07. This means it makes sense to keep the two separate DAO methods.